src/Microsoft.DotNet.Tools.Pack/Program.cs

```diff
@@ -51,7 +51,7 @@ public static int Main(string[] args)
                 var configValue = configuration.Value() ?? Cli.Utils.Constants.DefaultConfiguration;
                 var outputValue = output.Value();
 
-                return BuildPackage(path, configValue, outputValue, intermediateOutput.Value()) ? 1 : 0;
+                return BuildPackage(path, configValue, outputValue, intermediateOutput.Value()) ? 0 : 1;
```

Comment:
Hmm, maybe this should be a helper method?

`Microsoft.Internal.Helpers.ConvertReturnValueFromBooleanToIntegerForProcessExitCode` ? :trollface: